recipe/meta.yaml

```diff
@@ -24,13 +24,23 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - cmake
     - ninja
   host:
     - libqdldl ={{ qdldlversion }}
+    - numpy # For unittests
+    - scipy =1.11.4 # For unittests, as of 0.6.3 imports randn from scipy
 
 test:
```

Comment:
Please move these to the test section. The `*` is to avoid generating several builds for several numpy/python versions.
```suggestion

test:
  requires:
    - numpy *
    # 0.6.3 imports randn from scipy
    - scipy <1.12
```

Reply:
Unfortunately they are used in the build process to generate the unit test code. I have added numpy to ignore_run_exports, do I need to do more to ensure they stay as build only dependencies?

Reply:
I just realised that given numpy and scipy is used to generate code during build time but not something the package is built against, I guess they should be `build` dependencies not `host` dependencies?